modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -47,6 +58,99 @@ test.afterEach(async ({notificationTemplatesPage, page}) => {
 	}
 });
 
+test('Can use all notification terms in the rich text and check if the content is being persisted', async ({
+	emailNotificationTemplatePage,
+	notificationTemplatesPage,
+	page,
+}) => {
+	await emailNotificationTemplatePage.goto();
+
+	const notificationTemplateName =
+		'Notification Template Name' + getRandomInt();
+
+	await emailNotificationTemplatePage.basicInfoName.fill(
+		notificationTemplateName
+	);
+
+	await emailNotificationTemplatePage.senderEmailAddress.fill(
+		'test@liferay.com'
+	);
+
+	await emailNotificationTemplatePage.senderName.fill('test user');
+
+	await page.getByText('Current User Email Address').click();
+
+	await page.keyboard.press('Tab');
+
+	await page.keyboard.press('Enter');
```

Comment:
can you please explain why you'd to do this?

Reply:
i see...this is not bad and we can stay with this approach, although i think there is a more fitting way to do this.

it would take some time to search more in the documentation and also see if we would need to change the `aria-label` to be more specific or use something like an `id_test` for this

Reply:
I was also thinking a little about this, and how we are building these buttons from the fds itemsActions:

```
itemsActions={[
	{
		href: 'copyObjectFieldTerm',
		id: 'copyObjectFieldTerm',
		label: Liferay.Language.get('copy'),
		target: 'event',
	},
]}
```

I don't know if we have a clean and easy way to relate the term information to the button information. But I know that this is something very important and that we need map it, when we press the Tab key the screen reader  read what is in the aria-label, and the user will hear "copy" several times, without knowing which term is being copied.